packages/suite/src/views/wallet/send/components/Outputs/index.tsx

```diff
@@ -35,27 +40,71 @@ const Row = styled.div`
+    disableAnim?: boolean; // used in tests, with animations enabled react-testing-library can't find output fields
+}
+
+const Outputs = ({ disableAnim }: Props) => {
     const { outputs } = useSendFormContext();
+    const [renderedOutputs, setRenderedOutputs] = useState(1);
+    const lastOutputRef = useRef<HTMLDivElement | null>(null);
+
+    const onAddAnimationComplete = () => {
+        // scrolls only on adding outputs, doesn't scroll on removing them
+        if (outputs.length > 1 && outputs.length > renderedOutputs) {
+            lastOutputRef?.current?.scrollIntoView({ behavior: 'smooth' });
+        }
+
+        setRenderedOutputs(outputs.length);
+    };
+
+    useEffect(() => {
+        if (outputs.length < renderedOutputs) {
+            // updates rendered outputs count when removing an output
+            // this is necessary because onAddAnimationComplete is not fired when removing 2nd output
+            setRenderedOutputs(outputs.length);
+        }
+    }, [outputs.length, renderedOutputs, setRenderedOutputs]);
+
+    const WrapperComponent = disableAnim ? Wrapper : AnimatePresence;
     return (
-        <Wrapper>
-            {outputs.map((output, index) => (
-                <OutputWrapper key={output.id} index={index}>
-                    {output.type === 'opreturn' ? (
-                        <OpReturn outputId={index} />
-                    ) : (
-                        <>
-                            <Row>
-                                <Address outputId={index} outputsCount={outputs.length} />
-                            </Row>
-                            <Row>
-                                <Amount outputId={index} />
-                            </Row>
-                        </>
-                    )}
-                </OutputWrapper>
-            ))}
-        </Wrapper>
+        <WrapperComponent initial={false}>
+            {outputs.map((output, index) => {
+                const outputComponent = (
+                    <OutputWrapper
+                        ref={index === outputs.length - 1 ? lastOutputRef : undefined} // set ref to last output
+                        index={index}
+                        key={output.id}
+                    >
+                        {output.type === 'opreturn' ? (
+                            <OpReturn outputId={index} />
+                        ) : (
+                            <>
+                                <Row>
+                                    <Address outputId={index} outputsCount={outputs.length} />
+                                </Row>
+                                <Row>
+                                    <Amount outputId={index} />
+                                </Row>
+                            </>
+                        )}
+                    </OutputWrapper>
+                );
+
+                return disableAnim ? (
+                    outputComponent
+                ) : (
+                    <motion.div
```

Comment:
you can still use only one nesting (OutputWrapper), just change to `styled(motion.div)` and disable animation for 1 output and tests as well

Reply:
i got an idea to move this animated wrapper to `/send/components` and use it in multiple places, like opening/closing additional options (locktime, data etc.)

Reply:
If I try to do OutputWrapper = styled(motion.div) I am getting a runtime error `Warning: React has detected a change in the order of Hooks called by ForwardRef(MotionComponent)....`, not sure what it doesn't like (maybe passing a ref?). At least I removed condition in return as it was not necessary.

Reply:
I am all for adding animation to multiple places, but I would like this PR get merged before Friday as it addresses all requested improvements. Could we do that in another PR?